app/dashboard/static/js/app/view-tests-suite-id.2017.7.2.js

```diff
@@ -0,0 +1,557 @@
+    gFileServer = null;
+
+    // TODO Add a row from /jobs to tests ?
+
+    function addCaseTableRow(data, docFrag) {
+        var cellNode;
+        var logsNode;
+        var rowNode;
+        var caseName;
+        var definitionURI;
+        var logsNode;
+
+        caseName = data.name;
+        definitionURI = data.definition_uri;
+
+        rowNode = docFrag.appendChild(document.createElement('tr'));
+
+        // Name.
+        cellNode = rowNode.insertCell(-1);
+        cellNode.className = 'name-column';
+        cellNode.appendChild(document.createTextNode(caseName));
+
+        // Failure desc.
+        cellNode = rowNode.insertCell(-1);
+        cellNode.className = 'failure-column';
+        if (definitionURI) {
+            cellNode.appendChild(definitionURI);
+        } else {
+            cellNode.insertAdjacentHTML('beforeend', '&nbsp;');
+        }
+        // Test Case log.
+        // TODO Create Log from something
+        //logsNode = tboot.createBootLog();
+        cellNode = rowNode.insertCell(-1);
+        cellNode.className = 'pull-center';
+        //cellNode.appendChild(logsNode);
+        cellNode.insertAdjacentHTML('beforeend', '&nbsp;');
+
+        // Date.
+        cellNode = rowNode.insertCell(-1);
+        cellNode.className = 'date-column pull-center';
+        cellNode.appendChild(ttest.dateNode(data.created_on));
+
+        // Status.
+        cellNode = rowNode.insertCell(-1);
+        cellNode.className = 'pull-center';
+        cellNode.appendChild(ttest.statusNode(data.status));
+
+        // Detail.
+        cellNode = rowNode.insertCell(-1);
+        cellNode.className = 'pull-center';
+        cellNode.appendChild(ttest.detailsNode(data));
+    }
+
+    function getMultiCasesDataFail() {
+        // TODO
+    }
+
+    function getMultiCasesDataDone(response) {
+        var docFrag;
+        var results;
+        var table;
+        var tableId;
+        var tableBody;
+
+        docFrag = document.createDocumentFragment();
+        // Because we have an aggregate of deferred
+        results = response[0].result;
+
+        // For each test case create a table row
+        // And update the matching set table
+        results.forEach(function(result) {
+            addCaseTableRow(result, docFrag)
+
+            if(result.test_set_id) {
+                tableId = 'table-set-' + result.test_set_id.$oid;
+                table = document.getElementById(tableId);
+            } else {
+                // Cases that doesn't belong to a test set
+                tableId = 'multiple-cases-table';
+            }
+
+            table = document.getElementById(tableId);
+            html.removeClass(table, 'hidden');
+            table.tBodies[0].appendChild(docFrag);
+        });
+
+        if (results.length === 0) {
+            tableId = 'multiple-cases-table';
+            table = document.getElementById(tableId);
+            setTimeout(
+                _tableMessage.bind(
```

Comment:
Where is this function defined?

Reply:
Missed the hunk during the add, it's there now: https://github.com/loysollivier/kernelci-frontend/blob/fa78179f546b2ba864b3695d73426264cdcab211/app/dashboard/static/js/app/view-tests-suite-id.2017.7.2.js#L46